features/debug-tools/config.mk

```diff
@@ -1,2 +1,2 @@
 DEBUG_TOOLS_PROGS := gdb strace
-DEBUG_TOOLS_FILES := $(DATADIR)/gdb
+DEBUG_TOOLS_FILES := $(DATADIR)/gdb $(LIBDIR)/terminfo/ $(DATADIR)/terminfo/
```

Comment:
Another problem is that only on systemd-based installations `/lib` and `/usr/lib` are the same thing.

```
ls -1d {,/usr}/{share,lib{,64}}/terminfo
ls: cannot access '/share/terminfo': No such file or directory
ls: cannot access '/lib64/terminfo': No such file or directory
ls: cannot access '/usr/lib/terminfo': No such file or directory
ls: cannot access '/usr/lib64/terminfo': No such file or directory
/lib/terminfo
/usr/share/terminfo
```
You do not need to add `$(LIBDIR)/terminfo/` at all. The terminfo database has symlinks in / lib if needed.

Reply:
fixed